tests/data/simpleFramerISD.json

```diff
@@ -1,96 +1,54 @@
 {
-    "boresight": [
-        0.0,
-        0.0,
-        1.0
-    ],
-    "ccd_center": [
-        7.5,
-        7.5
-    ],
-    "ephemeris_time": 100.0,
-    "focal_length": 50,
-    "focal_length_epsilon": 1.0,
-    "ifov": 6.0,
-    "model_name": "USGS_ASTRO_FRAME_SENSOR_MODEL",
-    "spacecraft_name": "TEST_CRAFT",
-    "instrument_id": "TEST_SENSOR",
-    "target_name": "TEST_BALL",
-    "pixel_pitch": 0.1,
-    "itrans_line": [
-        0.0,
-        0.0,
-        10
-    ],
-    "itrans_sample": [
-        0.0,
-        10,
-        0.0
-    ],
-    "transx": [
-        0.0,
-        0.1,
-        0.0
-    ],
-    "transy": [
-        0.0,
-        0.0,
-        0.1
-    ],
-    "min_elevation": -1,
-    "max_elevation": 1,
-    "nlines": 16,
-    "nsamples": 16,
-    "original_half_lines": 8.0,
-    "original_half_samples": 8.0,
-    "omega": 0,
-    "phi": -1.5707963267948966,
-    "kappa": 3.141592653589793,
-    "semi_major_axis":0.01,
-    "semi_minor_axis":0.01,
-    "transx": [
-        0.0,
-        0.1,
-        0.0
-    ],
-    "transy": [
-        0.0,
-        0.0,
-        0.1
-    ],
-    "x_sensor_origin": 1000,
-    "y_sensor_origin": 0,
-    "z_sensor_origin": 0,
-    "x_sensor_velocity": 1,
-    "y_sensor_velocity": 0,
-    "z_sensor_velocity": 0,
-    "x_sun_position": 100,
-    "y_sun_position": 100,
-    "z_sun_position": 0,
-    "odt_x": [
-        0.0,
-        1.0,
-        0.0,
-        0.0,
-        0.0,
-        0.0,
-        0.0,
-        0.0,
-        0.0,
-        0.0
-    ],
-    "odt_y": [
-        0.0,
-        1.0,
-        0.0,
-        0.0,
-        0.0,
-        0.0,
-        0.0,
-        0.0,
-        0.0,
-        0.0
-    ],
-    "starting_detector_line": 0.0,
-    "starting_detector_sample": 0.0
-}
+	"center_ephemeris_time": 50,
+	"dt_ephemeris": 100,
+	"focal2pixel_lines": [0.0, 0.0, 0.1],
+	"focal2pixel_samples": [0.0, 0.1, 0.0],
```

Comment:
These parameters should be focal plane x, y to pixel conversions. These values are pixels to focal plane x, y

Reply:
In the plugin these values are being put in the pixel to focal plane coefficient vectors, so it's a double negative situation.